### DISCUSSION TITLE
Add Web overlay on streaming.

### DISCUSSION THREAD

**Discussion by RC7770** [HaishinKit/HaishinKit.swift#1480] [Q&A]
I recently utilized the Advanced Video Editing feature #[[1449](https://github.com/shogo4405/HaishinKit.swift/pull/1449)], and I must say, I was impressed by its functionality. Building upon that experience, I am now seeking your expertise to help me develop a similar feature for my live-streaming application.

Specifically, I want to integrate a Score ticker for live matches into my streaming platform. Currently, I am using the VideoEffect class, but it falls short of meeting all my requirements. I envision creating a class, perhaps named WebScreenObject, that functions similarly to VideoTrackScreenObject or AssetScreenObject but is specifically tailored for WKWebView integration.

Your assistance in developing this feature would be invaluable to me, and I greatly appreciate any guidance or support you can provide.

Thank you for considering my request.

**Comment by shogo4405** [comment]
What would you like me to help with? I believe it can be done in about two days. I charge 800 USD per day, so I can assist you for a total of 1,600 USD for two days. Please consider this offer.

Please try it yourself first. If it doesn't work, please share the code. I can't provide advice without specific details.

**Reply by shogo4405** [reply]
After checking the source code, I realized that it should be open instead of public so that it can be inherited and used from the application layer. This was not intentional, so I will fix it.

https://github.com/shogo4405/HaishinKit.swift/blob/main/Sources/Screen/ScreenObject.swift#L17-L134
